tools/cell_census_builder/tests/test_builder.py

```diff
@@ -46,15 +44,21 @@ def setup_class(cls) -> None:
         process_initializer()
 
         # The tile extent needs to be smaller than the default (2048) to work with the fixture
-        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_0"]["tile"] = 2  # type: ignore
-        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_1"]["tile"] = 2  # type: ignore
+        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_0"]["tile"] = 2
+        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_1"]["tile"] = 2
 
     @classmethod
     def teardown_class(cls) -> None:
         cls.td.cleanup()
 
-    @pytest.fixture
-    def h5ad(self) -> anndata.AnnData:
+    @dataclass
```

Comment:
suggest `@dataclass(frozen=True)`

Reply:
also, nit:  have been migrating from dataclass to attrs based upon Paul's recommendation. It is definintely an improvement in some ways.

Reply:
good call, done.  we have 2 other dataclasses still in the code base, but I'm leaving those alone for now w/TODOs